packages/flutter/lib/src/material/list_tile.dart

```diff
@@ -1039,7 +1039,7 @@ class _ListTileElement extends RenderObjectElement {
     _updateChild(widget.trailing, _ListTileSlot.trailing);
   }
 
-  void _updateRenderObject(RenderObject child, _ListTileSlot slot) {
+  void _updateRenderObject(RenderBox child, _ListTileSlot slot) {
```

Comment:
Was this intentional?

This method appears to be just fine with a `RenderObject`, and it introduces an additional cast below.

Reply:
`_ListTileSlot` is an enum with the 4 values managed by the switch bellow.

In every cases of the `switch` bellow `child` is assign as a `RenderBox` so it made sence to change the type of the parameter (and avoid 4 `as RenderBox`).

Reply:
Ahhh ok, thanks for clearing that up.